### DISCUSSION TITLE
IRC Channel

### DISCUSSION THREAD

**Discussion by Feoramund** [odin-lang/Odin#3834] [General]
For people who like IRC, I made a channel at `#odin` on https://www.hackint.org/ for casual chatting or technical support.

**EDIT**: Right, if you're new to IRC, you'll want to register with `NickServ`. You can get a list of commands by sending `/msg nickserv help`. You'll also want to take the default HackInt cloak by messaging `HostServ` after you're done with that, so you can hide your IP from others: `/msg hostserv take hackint/user/$account`.

Other than that, any usual IRC guide should help you along.

**Comment by Kelimion** [comment]
I've registered an account on HackInt (w/ NickServ).
